ZenPacks/zenoss/ZenPackLib/tests/ZPLTestBase.py

```diff
@@ -38,6 +44,13 @@ class ZPLTestBase(BaseTestCase):
     disableLogging = True
     z = None
 
+    def setUp(self):
+        if not run_tests:
+            msg = 'Skipping {} because environment '\
```

Comment:
Just a style thing: try to avoid backslash line-continuations. Especially this case where the continuation isn't even indented. I think something like the following reads better, and style checkers are usually happier with.

```python
msg = (
    "Skipping {} because environment "
    "variable RUN_ZPL_TESTS is unset."
    ).format(self.__class__.__name__)
```